src/components/PhotoImporter/PhotoGallery.js

```diff
@@ -185,6 +185,7 @@ const PhotoGallery = ( ): Node => {
       navToObsEdit( );
       return;
     }
+
     navigation.navigate( "GroupPhotos", { selectedPhotos } );
```

Comment:
not sure if this is an issue specific to this PR, but I noticed that the number of observations never updates on the GroupPhotos screen when combining, separating, or deleting photos. And number of photos is also incorrect.

Reply:
Yes I noticed that too when I first worked on this. I wasn't exactly sure and didnt look into how they were calculated